docs/backends.rst

```diff
@@ -0,0 +1,211 @@
+===================================
+ Proctoring backend implementation
+===================================
+
+Proctoring providers (PS) who wish to integrate with Open edX should implement a `REST API`_ and a thin `Python wrapper`_, as described below.
+
+REST API
+--------
+
+Implement the following endpoints. In order to authenticate requests from the Open edX server, the PS backend should
+enable Oauth 2 authentication using JWT_. The Open edX installation will be configured with a client key and a client secret provided by the PS and will authenticate each request to the PS.
+
+To obtain a JWT_ token, the PS system must make an Oauth2 request to ``/oauth2/access_token``.
+
+SSL is required for all (production environment) requests.
+
+All requests and responses in this API are formatted as JSON objects.
+
+
+Proctoring System configuration endpoint
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    /v1/config/
+
+``GET``: returns an object of the available configuration options and metadata for the proctoring service.::
+
+    {
+        "config": {
+            "allow_multiple": "Allow multiple monitors",
+            "allow_notes": "Allow paper notes",
+            "allow_apps": "Allow other applications to be running",
+            ...
+        },
+        "name": "My Proctoring Provider",
+        "download_url": "https://my.provider.com/download/software",
+        "instructions": [
+            "Step one, download the software",
+            "Next, run the software",
+            ...
+        ]
+    }
+
+The keys in the config object should be machine readable. The values are human readable. PS should respect the HTTP request ``Accept-Language``
+header and translate all human readable configuration options into the requested language, if possible. 
```

Comment:
Just wanted to confirm we're confident that Verificient can handle this i18n subtlety for us. Do we need to raise this in a communication with them?